src/app/public/modules/i18n/resource-name-provider.spec.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  SkyAppResourceNameProvider
+} from './resource-name-provider';
+
+describe('Resource name provider', () => {
+  it('should return the input resouce name', () => {
```

Comment:
I would add the `done` parameter and call it after the `expect`.  I believe it's passing since your mock is immediately returning `Observable.of()`, but it still feels a little weird to have a `.subscribe()` without the done.

Reply:
I should clarify, I don't actually know if this would pass.  Since this is running from a fork, we don't run the full test suite.  Let's have a quick chat offline about how we can fix that.